### DISCUSSION TITLE
How can I apply for the Github Education Package with a district-locked school email address?

### DISCUSSION THREAD

**Discussion by SpyGuy0215** [community/community#21254] [GitHub Education] [answered]
My district uses an email that can only receive emails from other people in the district, meaning that I can't use a verification email sent to that address to verify my account. I really need this because I want to keep using Github Copilot even after the technical preview ends, so how would I do that? I meet all other requirements, so this is quite a shame. Thanks in advance for any answers!

**Comment by baysideengineer** [comment, marked as answer]
Hello @SpyGuy0215, I am not sure if you really need your email address for verification. In the [documentation](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/use-github-for-your-schoolwork/apply-for-a-student-developer-pack) and also on the [application page](https://education.github.com/discount_requests/student_application) there is one line that might be interesting.

> Have a verifiable school-issued email address or upload documents that prove your current student status.

The interesting word here might be the **or**. There is also another interesting note.

> Selecting a school-issued email address gives you the best chance of a speedy review.

I am not sure if you have tried yet to enroll but I would suggest go ahead and apply for your [student developer pack](https://education.github.com/discount_requests/student_application) and see how things are going. If some problems come up, then you can contact the [GitHub Education support](https://support.github.com/contact/education) and ask for help. I am sure they can help you. Wish you good luck with your application.

**Comment by c9long** [comment]
You can use a personal email address (for example, one that ends in @gmail.com) and apply by supplying documents as @saschazengler has already suggested.

I must warn you in advance that their system for document verification is quite picky. Be prepared to have multiple proofs of enrolment, in paper, containing a clearly visible date between 2018 and 2026, with your school's name and not just a logo, so you can take full-page photos of any of your IDs with a device camera, lest you run into any of the following errors:
![image](https://user-images.githubusercontent.com/109810526/181135656-1314903b-3d14-43a9-9341-ad5506783023.png)
![image](https://user-images.githubusercontent.com/109810526/181135743-02e77a0a-cfae-494e-ac99-0e94a5d8927e.png)

**Comment by SpyGuy0215** [comment]
Turns out that my school district is on the list, but I have to create a new one just to bypass the "required school email" issue. Thanks for all the help!
